daemon/daemon.go

```diff
@@ -151,7 +151,6 @@ func (daemon *Daemon) restore() error {
 			container.RWLayer = rwlayer
 			if err := daemon.Mount(container); err != nil {
 				logrus.Errorf("Failed to mount container %v: %v", id, err)
-				continue
```

Comment:
perhaps we should add a comment here to explain why we don't "continue", otherwise someone may "fix" it in future